art/dashboard/app.py

```diff
@@ -0,0 +1,182 @@
+import json
+from collections import defaultdict
+from pathlib import Path
+
+import dash_bootstrap_components as dbc
+import dash_mantine_components as dmc
+import pandas as pd
+from dash import Dash, Input, Output, callback, dash_table, dcc, html
+
+app = Dash(__name__, external_stylesheets=[dbc.themes.BOOTSTRAP])
+
+LOGS_PATH = Path("ideal_logs")
+
+steps = defaultdict(lambda: {"full_path": [], "id": [], "results": None, "log": ""})
+
+
+def merge_steps(step1_dict, step_2_dict):
```

Comment:
Just to notice, there's no typing - but for now it's not that important

Reply:
I added it